pkg/smokescreen/conntrack/conn_tracker.go

```diff
@@ -155,3 +164,7 @@ func (tr *Tracker) MaybeIdleIn(d time.Duration) time.Duration {
 	})
 	return longest
 }
+
+func (tr *Tracker) Wg() *sync.WaitGroup {
```

Comment:
While you're thinking about whether `*Tracker` implements `TrackerInterface`, you might want to have the compiler check for you; something like `var _ TrackerInterface = &Tracker{}` at the bottom of this file would do that.